src/Schema/SchemaValidator.php

```diff
@@ -57,7 +57,7 @@ public function validate($data, CebeSchema $schema, ?BreadCrumb $breadCrumb = nu
 
         try {
             // These keywords are not part of the JSON Schema at all (new to OAS)
-            (new Nullable($schema))->validate($data, $schema->nullable);
+            (new Nullable($schema))->validate($data, $schema->nullable ?? true);
```

Comment:
(I didn't understand why we want `nullable` `true` by default instead of `false` ?)